dev-libs/libutf8proc/libutf8proc-2.2.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+MY_PN="${PN/lib/}"
+
+inherit cmake-utils toolchain-funcs
+
+DESCRIPTION="A C library that provides various operations for data in the UTF-8 encoding"
+HOMEPAGE="https://juliastrings.github.io/utf8proc/"
+SRC_URI="https://github.com/JuliaStrings/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="test"
+
+# Tests do need internet access
+RESTRICT="test"
+
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+DOCS=(
```

Comment:
dodoc in src_install

Reply:
Ok, will do. Could you explain me, why? I don't see any difference.

Reply:
what is the point to define `DOCS` and then call its installation, `DOCS` is good once you are using default installation implementation (undeclared one), for the rest, it is pointless

Reply:
You could ask the same question in reverse ;-) Both achieve exactly the same, so IMHO it doesn't matter at all. I think, you could see faster the definied DOCS than somewhere in src_*. I would agree, when one would define DOCS and also some `dodoc`. Anyway, will have a eye on that.